modFastVM/test/org/aion/fastvm/FastVMTest.java

```diff
@@ -452,9 +454,11 @@ public void testBlockCreationExploit() throws InterruptedException {
         // try to connect the deployment block
         ImportResult result = bc.tryToConnect(context.block);
         assertThat(result).isEqualTo(ImportResult.IMPORTED_BEST);
-        AionAddress contractAddress = tx.getContractAddress();
+        Address contractAddress = tx.getContractAddress();
         System.out.println(
-                "xxx = " + bc.getRepository().getNonce(AionAddress.wrap(deployerAccount.getAddress())));
+                "xxx = "
```

Comment:
should we delete this

Reply:
Since they're inside a unit test, I would leave them for now. I'll add this to the clean-up to do list though.